### ISSUE TITLE
etcd-cpp-apiv3 build issue

### ISSUE THREAD

**Issue by chenrui333** [etcd-cpp-apiv3/etcd-cpp-apiv3#185]
👋 when trying to build etcd-cpp-apiv3 against [abseil 20230125.0 release](https://github.com/Homebrew/homebrew-core/pull/121718), we found some build error

```
/tmp/etcd-cpp-apiv3-20230128-51795-1p4oa76/etcd-cpp-apiv3-0.2.10/src/SyncClient.cpp:293:43: error: no viable conversion from 'shared_ptr<grpc_impl::Channel>' to 'const shared_ptr<::grpc::ChannelInterface>'
  stubs->watchServiceStub= Watch::NewStub(this->channel);
                                          ^~~~~~~~~~~~~
```

relates to https://github.com/Homebrew/homebrew-core/pull/121718
full build log, https://github.com/Homebrew/homebrew-core/actions/runs/4029482566/jobs/6927459144

**Comment by chenrui333** [comment]
grpc version that I am using is `1.51.1`

**Comment by chenrui333** [comment]
Any thoughts?

**Comment by sighingnow** [comment]
I would try with the latest grpc. Thanks for reporting.

**Comment by sighingnow** [comment]
Fixed in #187 and a new release (v0.2.11) has been posted.

**Comment by sighingnow** [comment]
> Fixed in #187 and a new release (v0.2.11) has been posted.

See also: https://github.com/Homebrew/homebrew-core/pull/121899

**Comment by chenrui333** [comment]
NICE!! Thanks @sighingnow!
